tests/thread_tests/remote.go

```diff
@@ -82,29 +82,32 @@ func connectSSH(t *testing.T) {
 	t.Logf("SSH: connected to %s", remoteHost)
 }
 
-func remote_deployOTBRAgent(t *testing.T) error {
+func remote_deployOTBRAgent(t *testing.T) {
 
 	t.Cleanup(func() {
 		remote_exec(t, "sudo snap remove --purge openthread-border-router")
 	})
 
+	start := time.Now().UTC()
+
 	commands := []string{
 		"sudo snap remove --purge openthread-border-router",
 		"sudo snap install openthread-border-router --edge",
 		"sudo snap set openthread-border-router infra-if='" + remoteInfraInterface + "'",
-		"sudo snap connect openthread-border-router:avahi-control",
+		// "sudo snap connect openthread-border-router:avahi-control",
 		"sudo snap connect openthread-border-router:firewall-control",
 		"sudo snap connect openthread-border-router:raw-usb",
 		"sudo snap connect openthread-border-router:network-control",
-		"sudo snap connect openthread-border-router:bluetooth-control",
-		"sudo snap connect openthread-border-router:bluez",
+		// "sudo snap connect openthread-border-router:bluetooth-control",
+		// "sudo snap connect openthread-border-router:bluez",
```

Comment:
```suggestion
```
Same as before.

Reply:
Could be removed. I am keeping them to make it clear these interfaces were available, but skipped intentionally. It doesn't affect the readability of the code. A future version of the test may need them.